docs/content/docs/connectors/datastream/firehose.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Firehose
+weight: 5
+type: docs
+aliases:
+- /dev/connectors/firehose.html
```

Comment:
not sure how this works, but should we add the below like kinesis data streams docs?
```
  - /apis/streaming/connectors/firehose.html
```

Reply:
I think this might be a legacy thing, not too sure tbh